ergo/distributions/point_density.py

```diff
@@ -217,39 +221,57 @@ def normalize_fixed_params(self, fixed_params, scale):
     # Export
 
     @classmethod
-    def add_endpoints(cls, xs, densities):
+    def add_endpoints(cls, xs, densities, scale=None):
         """
-        Assumes xs are normalized. Returns a list of xs and densities
-        with endpoints that are on the edge of the scale.
+        Returns a list of xs and densities with endpoints that are on the edge of the scale
+        provided. If no scale is provided assume data is normalized.
         """
-        if xs[0] != 0:
+        if scale is None:
+            scale = Scale(0, 1)
```

Comment:
Is this case used, or can we always require a `scale`?

Reply:
we can always require scale